README.md

```diff
@@ -218,6 +221,46 @@ Options:
 - `bitLength`: Number of id bits. (default: 128)
 - `multibase`: Account for multibase encoding. (default: true)
 
+### `generateKeySeed(options)`
```

Comment:
Thinking about this more, let's call this `generateSecretKeySeed` to make sure people may be more inclined to be careful with this. Same goes for `decodeSecretKeySeed`.

Reply:
Renamed .